service/worker/scheduler/workflow.go

```diff
@@ -760,6 +760,7 @@ func (s *scheduler) handleDescribeQuery() (*schedspb.DescribeResponse, error) {
 	// this is a query handler, don't modify s.Info directly
 	infoCopy := *s.Info
 	infoCopy.FutureActionTimes = s.getFutureActionTimes(false, s.tweakables.FutureActionCount)
+	infoCopy.BufferSize = int64(len(s.State.BufferedStarts))
```

Comment:
@dnr this is a very simple change, I was not sure if unit test is needed. lmk if you think otherwise.